packages/babel-preset-udemy-website/tests/result/desktop/arrows.js

```diff
@@ -8,9 +8,7 @@ import "core-js/modules/es6.array.from";
 import "core-js/modules/es7.array.includes";
 import "core-js/modules/es6.array.iterator";
 import "core-js/modules/es6.array.of";
-import "core-js/modules/es6.array.sort";
```

Comment:
We're targeting a *slightly* narrower set of desktop browsers now, so the following polyfills got dropped:
* core-js/modules/es6.array.sort
* core-js/modules/es6.date.to-json
* core-js/modules/es6.object.set-prototype-of

This is okay because all of these methods are supported by all browsers anyway.